environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/response/CredentialViewResponse.java

```diff
@@ -0,0 +1,82 @@
+package com.sequenceiq.environment.api.v1.credential.model.response;
+
+import static com.sequenceiq.environment.api.doc.ModelDescriptions.CRN;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.environment.api.doc.ModelDescriptions;
+import com.sequenceiq.environment.api.doc.credential.CredentialDescriptor;
+import com.sequenceiq.environment.api.doc.credential.CredentialModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = CredentialDescriptor.CREDENTIAL_VIEW, value = "CredentialViewV1Response")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CredentialViewResponse {
+
+    @ApiModelProperty(ModelDescriptions.NAME)
+    private String name;
+
+    @ApiModelProperty(CRN)
```

Comment:
the static import here is not consistent with other constants

Reply:
ok